torch/csrc/distributed/rpc/init.cpp

```diff
@@ -250,6 +250,7 @@ PyObject* rpc_init(PyObject* _unused, PyObject* noargs) {
           .def(
               "to_here",
               &PyRRef::toHere,
+              py::arg("device_map") = DeviceMap(),
```

Comment:
What's the default behavior here? Will it inherit the global device map or fallback to CPU RPC?